.github/workflows/continuous-integration.yml

```diff
@@ -321,34 +321,9 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Install dependencies
-        uses: ./.github/workflows/install
```

Comment:
Are we planning on making a global action for the `install` composite action?

Reply:
not for now but open to have a conversation on this. I noticed other composite actions are calling this and I am thinking this might be going back n forth for the effort of calling an action.

Reply:
Yeah, I'm also not sure that you can even call a composite action inside of another one